lua/octo/utils.lua

```diff
@@ -364,17 +364,41 @@ function M.checkout_pr_sync(pr_number)
   }):sync()
 end
 
----Mergest a PR b number
+function M.insert_merge_flag(args, method)
+  if method == "squash" then
+    table.insert(args, "--squash")
+  elseif method == "rebase" then
+    table.insert(args, "--rebase")
+  else
+    table.insert(args, "--merge")
+  end
+end
+
+---Merges a PR by number
 function M.merge_pr(pr_number)
   if not Job then
+    M.error "Aborting PR merge"
     return
   end
+
+  local conf = config.values
+
+  local args = { "pr", "merge", pr_number, "--delete-branch" }
```

Comment:
since `--delete-branch` isn't added in `Octo pr merge` by default maybe we should add a setting for it and apply it by default in both cases?

Reply:
Agreed, please add a new setting and apply it consistently

Reply:
added `default_delete_branch`